x-pack/solutions/security/plugins/security_solution/public/flyout/shared/components/flyout_history_row.tsx

```diff
@@ -94,6 +98,28 @@ export const FlyoutHistoryRow: FC<FlyoutHistoryRowProps> = memo(({ item, index }
           dataTestSubj={NETWORK_HISTORY_ROW_TEST_ID}
         />
       );
+    case MisconfigurationFindingsPanelKey:
+      return (
+        <GenericHistoryRow
+          item={item}
+          index={index}
+          title={String(item?.panel?.params?.resourceId)}
+          icon={'document'}
+          name={'Resource Id'}
+          dataTestSubj={MISCONFIGURATION_HISTORY_ROW_TEST_ID}
+        />
+      );
+    case VulnerabilityFindingsPanelKey:
+      return (
+        <GenericHistoryRow
+          item={item}
+          index={index}
+          title={String(item?.panel?.params?.resourceId)}
+          icon={'document'}
+          name={'Resource Id'}
+          dataTestSubj={VULNERABILITY_HISTORY_ROW_TEST_ID}
+        />
+      );
```

Comment:
minor comment, but all the props except the `dataTestSubj` are identical so you could do something like this?
```typescript
case MisconfigurationFindingsPanelKey:
case VulnerabilityFindingsPanelKey:
      return (
        <GenericHistoryRow
          item={item}
          index={index}
          title={String(item?.panel?.params?.resourceId)}
          icon={'document'}
          name={'Resource Id'}
          dataTestSubj={TEST_ID}
        />
      );
```
If distinguishing between the 2 TEST_IDs is important maybe have a const value within the case?